src/simple_tracker_configuration/simple_tracker_configuration/configuration_node.py

```diff
@@ -35,8 +35,7 @@ def __init__(self, publisher_qos_profile: QoSProfile):
             self.get_config_array_callback)
         self.config_change_service = self.create_service(ConfigEntryUpdate, 'sky360/config/entry/update/v1', 
             self.get_config_update_callback)
-        self.config_change_publisher = self.create_publisher(ConfigEntryUpdatedArray, 'sky360/config/updated/v1', 
-            publisher_qos_profile)
+        self.config_change_publisher = self.create_publisher(ConfigEntryUpdatedArray, 'sky360/config/updated/v1', 10)#, publisher_qos_profile)
```

Comment:
Same here. If you want to change from `10` in the future that's a lot of typing.

Better to use a named constant.

Reply:
Same comment as above